### ISSUE TITLE
Touchpad zoom not smooth on Windows 10 with touch screen

### ISSUE THREAD

**Issue by sveine** [Leaflet/Leaflet#7476]
> The touchpad zooms on integer levels with touch=false, decimal with touch=true. With touch=true a two finger zoom in/out is OK on fast finger move. Tried Google Maps. It is very smooth on both touchpad, screen zooming and mouse - so it is somehow possible to make it work.

That are valuable details. Could you open separate issue for this problem?

_Originally posted by @johnd0e in https://github.com/Leaflet/Leaflet/issues/6896#issuecomment-783449701_

**Comment by sveine** [comment]
Maybe zoomSnap should be set to zero automatic i leaflet for touch. I think no one really want it to other than zero if everything else is working nice for touch device.  It may be overridden on touch devices manually.

**Comment by johnd0e** [comment]
Perhaps we we already had related discussions: https://github.com/Leaflet/Leaflet/search?q=touchpad&type=issues
Please check and link here those are relevant to subject issue.

**Comment by sveine** [comment]
> 
> 
> Perhaps we we already had related discussions: https://github.com/Leaflet/Leaflet/search?q=touchpad&type=issues
> Please check and link here those are relevant to subject issue.

I have not been digging deep into all, but seems that they are about touchpad (true for my mac). My Windows 10 laptop with touch-screen behaved different. (zoomSnap:0)

**Comment by johnd0e** [comment]
> but seems that they are about touchpad

There is "Touchpad zoom" in the title of the issue, so please clarify all required conditions.
May be get this template as a sample: https://github.com/Leaflet/Leaflet/issues/new?assignees=&labels=&template=bug_report.md

**Comment by sveine** [comment]
> 
> 
> > but seems that they are about touchpad
> 
> There is "Touchpad zoom" in the title of the issue, so please clarify all required conditions.
> May be get this template as a sample: https://github.com/Leaflet/Leaflet/issues/new?assignees=&labels=&template=bug_report.md

The title is correct. The problem is on a PC with touchpad AND touch screen. The TOUCHPAD is not working well. With zoomSnap:0 the Touchpad is very slow, without it is to fast and jumping/snapping to integer zoom levels.

**Comment by johnd0e** [comment]
> The title is correct

Good.
But please describe the issue in details.
1. What is default behavior on your system
   - touchpad zoom
   - touchscreen zoom
2. What is behavior with `zoomSnap:0`
   - touchpad zoom
   - touchscreen zoom
3. What changes after #7029 applied (https://gist.githack.com/johnd0e/3ad393c73b12b24af7e947711a87b7a4/raw/issue-7476.html)
   - touchpad zoom
   - touchscreen zoom

**Comment by Falke-Design** [comment]
Duplicate of #4410
